stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -2236,6 +2344,9 @@ impl NakamotoChainState {
 
         let vrf_proof_bytes = vrf_proof.map(|proof| proof.to_hex());
 
+        let signer_signature = serde_json::to_string(&header.signer_signature)
+            .expect("Unable to serialize signer signatures");
```

Comment:
This should not be fatal.  The node should instead reject the block by returning an error.

Reply:
Good catch!